### ISSUE TITLE
Orbs should be tied to a major version of Pulumi

### ISSUE THREAD

**Issue by njbmartin** [pulumi/circleci#27]
Until today, I was relying on the `pulumi/pulumi@1.2.0` as part of my circle ci config. When Pulumi 2.0 was released, my builds instantly failed. Turns out this was due to the `--yes` requirement in Pulumi 2.0.

Ideally, the release of a new major version should not have caused the "orb" to fail.

**Comment by chrsmith** [comment]
Yes, sorry about that. The root issue is that the `pulumi/login` command downloads the "latest" version of the CLI by default. You can set the `version` parameter to fix this to an older version of Pulumi to avoid any changes there. But having the orbs emit a warning and only get the latest major version (e.g. up to 2.x.x) would avoid breaking changes like this in the future.
